src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -167,20 +170,29 @@
             }
         }
 
+        .datagrid-column, .datagrid-cell {
+            text-align: left;
+            min-width: 4rem;
+            clr-icon {
```

Comment:
Is this targeting all icons inside of the datagrid-column and datagrid-cell? Even the ones projected by the user? If not, then can we use a class instead of the `clr-icon` tag?